chemprop/v2/cli/convert.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import torch
+import logging
+from pathlib import Path
+from argparse import ArgumentParser, Namespace
+
+from chemprop.v2.cli.utils import Subcommand
+from chemprop.v2.utils.v1_to_v2 import convert_model_v1_to_v2_file
+
+logger = logging.getLogger(__name__)
+
+class ConvertSubcommand(Subcommand):
+    COMMAND = "convert"
+    HELP = "convert a v1 model checkpoint (.pt) to a v2 model checkpoint (.ckpt)"
+
+    @classmethod
+    def add_args(cls, parser: ArgumentParser) -> ArgumentParser:
+        return add_convert_args(parser)
+
+    @classmethod
+    def func(cls, args: Namespace):
+        args = process_args(args)
+        main(args)
+
+def add_convert_args(parser: ArgumentParser) -> ArgumentParser:
+    parser.add_argument(
+        "-i",
+        "--input-path",
+        type=str,
+        required=True,
+        help="Path to a v1 model .pt checkpoint file.",
+    )
```

Comment:
```suggestion
    parser.add_argument(
        "input_path",
        help="Path to a v1 model .pt checkpoint file.",
    )
```
This makes `input-path` into a positional arg, which is always required. IMO it makes sense that a user calls this with only two arguments: (1) input path and (2, optional) output path